client/src/components/CreateInvoiceForm/index.js

```diff
@@ -247,6 +279,7 @@ export default class index extends Component {
   }
 
   render() {
+    // console.log(zipCodeTo,"render")
```

Comment:
Nice work! Might want to remove the console.log on the next refactor of this file